### PR TITLE
Added unit test for clocal-azure functions

### PR DESCRIPTION
**Pull request by kunalbhatia18**
I’ve created a unit testing pipeline for Azure functions allowing us to develop in a test-first style. In the process I have factored application logic out of the Azure Functions code nicely separating concerns of application logic vs. handling web requests.

### CODE REVIEW

**Comment by rajikaimal** [comment]
What's the use of `wrapper_test.js` file?

**Comment by rajikaimal** [comment]
And which logic / functionality in clocal is tested in `wrapper.js` file?

**Comment by kunalbhatia18** [comment]
> And which logic / functionality in clocal is tested in `wrapper.js` file?

i have used TDD to refactor some candidate code from the azure function file

**Comment by rajikaimal** [comment]
> i have used TDD to refactor some candidate code from the azure function file
If you're testing the logic in clocal, where's the import of any file in clocal-azure?

FYI, this is not TDD.

**Comment by rajikaimal** [comment]
> just to fix the path of the ava.js library

And you've removed it now, why is that?

**Comment by kunalbhatia18** [comment]
> > just to fix the path of the ava.js library
> 
> And you've removed it now, why is that?

i can set the path in other file too, no problem in that

**Comment by kunalbhatia18** [comment]
> > i have used TDD to refactor some candidate code from the azure function file
> 
> If you're testing the logic in clocal, where's the import of any file in clocal-azure?

that's why i have made another folder of testing we just have to set the path and you are ready for testing. instead of my eg in my readme.md put the function to test

**Comment by rajikaimal** [comment]
If I can understand correctly, according to your comments, you haven't yet added a unit test which test a certain functionality in clocal-azure. Is that so?

**Comment by kunalbhatia18** [comment]
> If I can understand correctly, according to your comments, you haven't yet added a unit test which test a certain functionality in clocal-azure. Is that so?

I added it on my PC but not here as the code was too long so i wanted to make my presentation a bit clean and small

**Comment by rajikaimal** [comment]
Well, then what's the point in this PR. We need a unit test, as the task clearly mentions. Add your unit tests to this PR.

**Comment by kunalbhatia18** [comment]
> Well, then what's the point in this PR. We need a unit test, as the task clearly mentions. Add your unit tests to this PR.

Done you can my readme.md I have provided the links

File: testing/readme.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Set path of tests in package.json in AVA to find the tests:
 Add some simple test code. 
 These parameters will be in the Azure Functions request object:
 
-    import sleeps from "./wrapper.js";
+    import sleeps from "./src/services/azure-functions/azure-function.js";
```
Comment (rajikaimal) [inline testing/readme.md]: You're trying to import sleeps. What is sleeps?

**Comment by rajikaimal** [comment]
How do you run unit tests if they are in a markdown file, we need .js files with the tests.

**Comment by rajikaimal** [comment]
Closing this as there are no unit tests implemented.

**Pull request closed without merging**